website/app/services/head-data.js

```diff
@@ -18,11 +18,8 @@ export default class CustomHeadDataService extends HeadDataService {
   }
 
   get imgSrc() {
-    return this.currentRouteMeta?.id
-      ? `/assets/illustrations/${this.currentRouteMeta?.id.replace(
-          /\/index$/,
-          ''
-        )}.jpg`
+    return this.currentRouteMeta?.frontmatter
+      ? this.currentRouteMeta.frontmatter.previewImage
```

Comment:
what if the `frontmatter` key exists, but the `previewImage` value is not defined? in that case the image is undefined. maybe we should do something like this?

```suggestion
    return this.currentRouteMeta?.frontmatter?.previewImage
      ? this.currentRouteMeta.frontmatter.previewImage
```